NEWS

```diff
@@ -1,3 +1,8 @@
+2017-02-17 Abhinav Rai <abhinavrai44@gmail.com>
+	* SHOGUN Release version 5.0.0 (libshogun 17.2, data 0.11, parameter 1)
+	* Features:
```

Comment:
You need to put just one line in the 5.1 here...no need to create a new release ;)
Also just say that we added Scala interface to the build, it's not like we added a new language ...